prebid-server/developers/add-new-bidder-go.md

```diff
@@ -990,6 +991,7 @@ func getBidMeta(bid *adapters.TypedBid) *openrtb_ext.ExtBidPrebidMeta {
     NetworkName:          "Some Network Name",
     PrimaryCategoryID:    "IAB-1",
     SecondaryCategoryIDs: []string{"IAB-2", "IAB-3"},
+    RendererUrl:          "https://example-renderer-url.com"
```

Comment:
@ccorbo - would it be fair to suggest that the example URL should point to a javascript file? `https://example-renderer-url.com/path/render.js` (?)

Reply:
Thanks Bret - updated to point to a js file